core/standards.md

```diff
@@ -0,0 +1,69 @@
+# Core Smart Contracts
+
+## Definition of Core
+
+A core contract is one that:
+
+1. Any contract that is part of the [dss repo](https://github.com/makerdao/dss)
+2. will have ward status on or other privileged access to any other core contract (i.e. `Vat.rely(<new contract address>)`)
+3. will rely on the Pause Proxy as a permissioned actor. If the pause proxy has to directly interact with this contract then there is additional security and process work that needs to be done.
+4. will be included in the chainlog as a canonical “Maker Contract”. Even if the contract does not have elevated permissions in the system, a failure of a contract in the chainlog could result in cost (bug bounties) or reputation damage for the protocol.
```

Comment:
```suggestion
4. will be included in the chainlog as a canonical “Maker Contract” (denoted by the prefix `MCD` in the chainlog key). Even if the contract does not have elevated permissions in the system, a failure of a contract in the chainlog could result in cost (bug bounties) or reputation damage for the protocol.
```

Reply:
I'm not sure it is always the case. We haven't been very consistent with the `MCD` tagged with important contracts such as moms, and including D3M.

Reply:
I thought it was referring to core contracts present in the dss repo

Reply:
Hmmm no, this point is not talking about `dss` contracts specifically.

Reply:
yeah in this case it is referring to "core" in the sense of a contract considered part of maker core protocol not just dss contracts. so like gonza said could be moms, d3ms, escrows, etc